net.sourceforge.pmd.eclipse.plugin/src/main/java/net/sourceforge/pmd/eclipse/runtime/properties/IProjectProperties.java

```diff
@@ -106,7 +107,7 @@
      * @return Returns the resolved RuleSet File suitable for loading a rule
      *         set.
      */
-    File getResolvedRuleSetFile() throws PropertiesException;
+    List<File> getResolvedRuleSetFile() throws PropertiesException;
```

Comment:
Since this returns now a List, I would expect a plural in the naming of the method, such as `getResolvedRuleSetFiles()`.